x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/service_overview/service_overview.spec.ts

```diff
@@ -204,17 +209,18 @@ describe('Service Overview', () => {
   describe('Calls APIs', () => {
     beforeEach(() => {
       cy.loginAsViewerUser();
-      cy.visit(baseUrl);
+
       apiRequestsToIntercept.map(({ endpoint, aliasName }) => {
         cy.intercept('GET', endpoint).as(aliasName);
       });
       apiRequestsToInterceptWithComparison.map(({ endpoint, aliasName }) => {
         cy.intercept('GET', endpoint).as(aliasName);
       });
+      cy.visitKibana(baseUrl);
     });
 
     it.skip('with the correct environment when changing the environment', () => {
-      cy.wait(aliasNames, { requestTimeout: 10000 });
+      cy.wait(aliasNames);
```

Comment:
Are we still skipping this tests?

Reply:
I'll leave it skipped for now. We should unskip in a follow-up PR.